src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -644,6 +741,31 @@ def __print_preview(old_json, new_json):
     return True
 
 
+def __print_restore_preview(kvs_to_restore, kvs_to_modify, kvs_to_delete):
+    print('\n---------------- Preview (Beta) ----------------')
+    if len(kvs_to_restore) + len(kvs_to_modify) + len(kvs_to_delete) == 0:
+        print('\nNo records matching found to be restored. No changes will be made.')
+        return False
+
+    # format result printing
+    if kvs_to_restore:
+        print('\nAdding:')
+        print(json.dumps(__serialize_kv_list_to_comparable_json_list(kvs_to_restore), indent=2))
+
+    if kvs_to_modify:
+        print('\nUpdating:')
+        print(json.dumps(__serialize_kv_list_to_comparable_json_list(kvs_to_modify), indent=2))
+
+    if kvs_to_delete:
+        print('\nDeleting:')
+        print(json.dumps(__serialize_kv_list_to_comparable_json_list(kvs_to_delete), indent=2))
+
+    print("")  # printing an empty line for formatting purpose
```

Comment:
Printing should all occur to STDERR. 
https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#general-patterns

Reply:
It make sense to print to stderr for error messages or logs. But in this case, we are trying to print out preview of a command. For example, to import some key-values into App Configuration, we would like to prompt a preview for confirmation. While, this preview can be a little complex, including key-values objects and some verbs marking "Updated, Deleted or Added". Should we still print to stderr for this scenario?

Reply:
Yes. The only thing that should go to STDOUT is JSON object structure that will obey CLI output format rules and can be programmatically ingested by other command line tools. This doesn't meet that standard. It's informational.

Reply:
@tjprescott is there a recommended way to write to STDERR? I've seen a few different ways people use but I'd want to be consistent with the rest of them.

Reply:
Changed the prints to error_print which now prints to sys,stderr